pywbem/_server.py

```diff
@@ -1024,6 +1042,49 @@ class variable.
         self._namespaces = [inst['Name'] for inst in ns_insts]
         self._namespace_paths = [inst.path for inst in ns_insts]
 
+        # An old version of a Hitachi server does not return its Interop
+        # namespace in the set of namespace instances. Pywbem adds the
+        # interop namespace to make up for that.
+        if interop_ns not in self._namespaces:
+
+            warnings.warn(
+                _format("Server at {0} has an Interop namespace {1!A}, but "
+                        "does not return it when enumerating class {2!A} "
+                        "- adding it",
+                        self.conn.url, interop_ns, ns_classname),
+                ToleratedServerIssueWarning, stacklevel=2)
+
+            self._namespaces.append(interop_ns)
+
+            # For adding the Interop namespace to the 'namespace_paths'
+            # property, its instance path is needed. Because the different
+            # classes used to represent namespaces have different keys, it
+            # seems best to copy one of their instance paths.
+            if not self._namespace_paths:
+                raise ModelError(
+                    _format("Server at {0} does not return the Interop "
+                            "namespace {1!A} when enumerating class {2!A}, "
+                            "and adding it to the list of namespaces is not "
+                            "possible because it does not have any namespaces "
+                            "besides the Interop namespace",
+                            self.conn.url, interop_ns, ns_classname))
+            org_path = self._namespace_paths[0]
+            interop_path = org_path.copy()
+            interop_path['Name'] = interop_ns
+            assert interop_path.namespace is not None
+            try:
+                self._conn.GetInstance(interop_path)
+            except CIMError as exc:
+                raise ModelError(
+                    _format("Server at {0} does not return the Interop "
+                            "namespace {1!A} when enumerating class {2!A}, "
+                            "and adding it to the list of namespaces is not "
+                            "possible because verification of the instance "
+                            "it via GetInstance on path {3!A} fails: {4}",
```

Comment:
I think the "it via" should be expanded to "it recieved via".

Reply:
Fixed by removing "it".